### PR TITLE
Add one requirement at the spotify integration page

### PR DESCRIPTION
**Pull request by N1co93**
## Proposed change

Add one requirement at the spotify integration page. This requirement isn't documented at the moment and would be really helpful for most of the people. See issue https://github.com/home-assistant/core/issues/40194

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: https://github.com/home-assistant/core/issues/40194

## Checklist

-  [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/spotify.markdown [hunk 1]
```diff
@@ -18,6 +18,7 @@ The `spotify` media player integration allows you to control [Spotify](https://w
 
 - Spotify account
 - Spotify application, properly configured (see below).
+- `internal_url` and `external_url` set correctly. See [Setup basic information](/docs/configuration/basic/)
```
Comment (frenck) [inline source/_integrations/spotify.markdown:21]: . This implies you need to set both, or is a requirement. This is not true.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/spotify.markdown (hunk 1 above)
Comment (N1co93) [reply source/_integrations/spotify.markdown:21]: You are right.

Maybe "and/or" would be the best way? In some situation both values have to be set. 

What do you think about it?

Comment (frenck) [reply source/_integrations/spotify.markdown:21]: I think in many cases no values should be set. The root cause lies in something else. If it was configured incorrectly, you should have gotten instruction from Home Assistant, which currently it does not.

I really think we should not add this.

Comment (N1co93) [reply source/_integrations/spotify.markdown:21]: Okay, then I will close this pull request.

**Pull request closed without merging**